style/index.css

```diff
@@ -36,4 +36,14 @@ li,
     font-weight: 300;
     margin-top: 0px;
     margin-bottom: 0px;
+}
+
+
+::-webkit-scrollbar { 
```

Comment:
Why do you use vendor prefix here?

Reply:
Ive seen this prefix on Stack Overflow as the best way to hide scroll bar

Reply:
And why do you hide scrollbar?

Reply:
It looks more beautiful🤘without scroll bars

Reply:
How then should the user understand that there is more content below and he need to scroll?

Reply:
page doesen't have 100vh image on the first screen which can stuck first user. Also when you see part of the elements below you always scroll down

Reply:
It's not intuitive

Reply:
first action which you do in the webpage? You don't see scroll bar but it doesen't matter you always scroll down

Reply:
How shold I know how long this page is and where am I?
How long should I scroll it?
How do I know where is the end?